redisinsight/api/src/modules/database-recommendation/scanner/strategies/abstract.recommendation.strategy.spec.ts

```diff
@@ -0,0 +1,11 @@
+import { DefaultRecommendationStrategy } from 'src/modules/database-recommendation/scanner/strategies';
+
+describe('AbstractRecommendationStrategy', () => {
+  const strategy = new DefaultRecommendationStrategy();
+
+  describe('isRecommendationReached', () => {
+    it('should get is recommendation reached', async () => {
+      expect(await strategy.isRecommendationReached()).toEqual(false);
```

Comment:
Well, now I see that default strategy should always return false. Will work. Anyway I prefer to have some log or telemetry in the future to understand that we have missed something

Reply:
Got it. I will think how better cover it by logger. Thanks